### PR TITLE
Add ninja install and usage to getting started docs

### PR DESCRIPTION
**Pull request by yakman2020**
@johnkord does this satisfy your concerns?

### CODE REVIEW

File: docs/GettingStartedDocs/Contributors/InstallInfo.md [hunk 1]
```diff
@@ -7,9 +7,14 @@ not currently need to be installed system-wide, so you could choose to install i
 into your home directory. From the build subfolder in your source tree:
 
 ```bash
-cmake -DCMAKE_INSTALL_PREFIX=~/openenclave-install ..
+cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=~/openenclave-install ..
```
Comment (johnkord) [inline docs/GettingStartedDocs/Contributors/InstallInfo.md]: Do you think we need to specify the "Unix Makefiles"?  I think by default it uses make in my experience, but I don't really know. @andschwa what do you think?

Comment (andschwa) [reply docs/GettingStartedDocs/Contributors/InstallInfo.md]: No, this is the default on Linux.

Comment (yakman2020) [reply docs/GettingStartedDocs/Contributors/InstallInfo.md]: @andschwa , adding "Unix Makefiles" is deliberate.  I'm trying to make a distinction between that and Ninja.  Besides, Unix Makefiles is not the default on windows (VC project is, I think) and that adds an additional degree of confusion.

Comment (yakman2020) [reply docs/GettingStartedDocs/Contributors/InstallInfo.md]: @andschwa Could I get an approval or a change request?

Comment (jxyang) [reply docs/GettingStartedDocs/Contributors/InstallInfo.md]: Yes, on windows the default target is Visual Studio

**Comment by yakman2020** [comment]
fixes #1947

**Pull request merged**